metron-interface/metron-rest/src/main/java/org/apache/metron/rest/service/PcapService.java

```diff
@@ -43,5 +44,5 @@
 
   InputStream getRawPcap(String username, String jobId, Integer page) throws RestException;
 
-
+  Map<String, Object> getConfiguration(String username, String jobId) throws RestException;
```

Comment:
Would be nice to see some javadocs here, but that doesn't seem unique to this PR.  Maybe in another PR we can get some javadocs in here.

Reply:
I added javadocs for this service.